### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by thabarbados**
### Fix or Enhancement?
Replacing vuepress with vitepress, update sandboxes and docs

- [x] All tests passed

### Environment
- OS: BigSur MacOs
- NPM Version: 8.5.1

### CODE REVIEW

**Comment by thabarbados** [comment]
Hey, @apertureless, could you update the docs site after merging this MR?

**Comment by apertureless** [comment]
@thabarbados The docs are hosted on github pages here.
You can just push it to the `gh-pages` branch and it will release it.

https://github.com/apertureless/vue-chartjs/blob/main/deploy-docs.sh

File: website/package.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "name": "website",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "docs:dev": "vitepress dev docs",
+    "docs:build": "vitepress build docs"
```
Comment (dangreen) [inline website/package.json:7]:
```suggestion
    "dev": "vitepress dev src",
    "build": "vitepress build src"
```

lets mv docs -> src

File: website/docs/guide/index.md [hunk 2]
```diff
@@ -0,0 +1,445 @@
...
+
+## Introduction
+
+`vue-chartjs` lets you use Chart.js without much hassle inside Vue. It's perfect for people who need simple charts up and running as fast as possible.
+
+It abstracts the basic logic but exposes the Chart.js object to give you maximal flexibility.
+
+## Installation
+
+### NPM
```
Comment (dangreen) [inline website/docs/guide/index.md:15]: useless title

File: website/docs/guide/index.md [hunk 3]
```diff
@@ -0,0 +1,445 @@
...
+
+```bash
+pnpm add vue-chartjs chart.js
+# or
+yarn add vue-chartjs chart.js
+# or
+npm i vue-chartjs chart.js
+```
+
+:::tip Need an API to fetch data?
```
Comment (dangreen) [inline website/docs/guide/index.md:27]: lets move it under introduction

**Review by dangreen — CHANGES_REQUESTED** [review]

File: website/docs/guide/index.md (hunk 2 above)
Comment (thabarbados) [reply website/docs/guide/index.md]: removed

File: website/docs/guide/index.md (hunk 3 above)
Comment (thabarbados) [reply website/docs/guide/index.md]: moved

**Review by dangreen — APPROVED** [review]

**Pull request merged**